DMTR-102.tex

```diff
@@ -44,183 +40,377 @@ \section{Introduction}
 
-            \multirow{1}{*}{ 4 } &  &
-            \begin{minipage}[t]{13cm}{\footnotesize
-            The LSP can review and use the CCOB raw data file that was stored originally somewhere else such as slac
+  503-8b raw image archiving for CCOB cycle\\
+  Status: Not Executed
 
-            \vspace{\dp0}
-            } \end{minipage} \\ \cline{2-3}
+  Test cycle for the raw image archiving of data from a CCOB device. ~This
+data needs to be stored at the LDF and ingested so that it can be looked
+at by scientists. ~ The data needs to be transferred to the LDF and then
+ingested into the archive service so that it can be retrieved through a
+butler to be displayed for further use. ~ This should be tested with a
+human involved at first for moving the data and making sure all the
+pieces work. ~ It should also be tested for when things are working more
+as a service, and without a human involved for an automatic data archive
+process. ~
 
-        \\ \midrule
-    \end{longtable}
 
+  \subsubsection{Software Version/Baseline}
+    The DBB version is 1.0, and can be found in the github repository when
+it has been fully developed. ~ The current DBB is a script for scraping
+data from the header file and getting the file name and the metadata
+required along with providence. ~ At the time of this writing, the DBB
+use cases and requirements are just being written. ~ ~The database
+component is Oracle and the ingest file systems and output file systems
+for DBB are in GPFS at the LDF.~~
 
 
-\newpage
+  \subsubsection{Configuration}
+    The configuration for this test is the data is located in a file system
+somewhere. ~It is transferred to NCSA into a the LSST GPFS file system.
+~ Files that are needed in the DBB are moved to the DBB ingest file
+system. ~ the data backbone gathers the info from the headers and file
+name. ~it inserts rows into the DBB consolidated DB environment and
+moves the file to the DBB file systems that are used by the LSP to once
+again find the raw image file to be viewed/displayed for further
+analysis. ~
 
-\section{Test Results}
-\label{sect:testresults}
 
-\subsection{Overview of the Test Results}
-\label{sect:overview}
+  \subsubsection{Test Cases in LVV-C8 Test Cycle}
 
-\subsubsection{Summary Table}
-\label{sect:summarytable}
 
+    \paragraph{Test Case LVV-T284 }\mbox{}\\
 
-\begin{longtable} {|p{0.2\textwidth}|p{0.2\textwidth}|p{0.6\textwidth}|}
-\hline
-{\bf TEST CASE ID} & {\bf PASS/FAIL} & {\bf COMMENTS} \\\hline
-LVV-T284 & Not Run & \\\hline
-\end{longtable}
+Open  {\it \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T284}{test case} } in Jira.
 
-\subsubsection{Overall Assessment}
-\label{sect:overallassessment}
+    This test will check:
 
+\begin{itemize}
+\tightlist
+\item
+  The successful integration of the Pathfinder components with the CCOB;
+\item
+  That the file can then be ingested into the DBB and be retrieved for
+  further analysis;
```

Comment:
Should not end in a `;`.

Reply:
This is defined in the test case. The corresponding test specification shall be reviewed.

Reply:
done